package.json

```diff
@@ -1,31 +1,75 @@
   "name": "grafana-mqtt-datasource",
-  "version": "1.0.0-beta.4",
+  "version": "1.1.0-beta.1",
   "description": "MQTT Datasource Plugin",
   "scripts": {
-    "build": "rm -rf dist && grafana-toolkit plugin:build && mage build:backend",
+    "build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
+    "dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
+    "lint": "eslint --cache --ignore-path ./.gitignore --ext .js,.jsx,.ts,.tsx .",
+    "lint:fix": "yarn run lint --fix",
+    "server": "docker-compose up --build",
+    "sign": "npx --yes @grafana/sign-plugin@latest",
     "spellcheck": "cspell -c cspell.config.json \"**/*.{ts,tsx,js,go,md,mdx,yml,yaml,json,scss,css}\"",
-    "test": "grafana-toolkit plugin:test && mage test",
-    "dev": "grafana-toolkit plugin:dev",
-    "watch": "grafana-toolkit plugin:dev --watch",
-    "sign": "grafana-toolkit plugin:sign",
-    "start": "yarn watch"
+    "start": "yarn dev",
+    "test": "jest --watch --onlyChanged",
+    "test:ci": "jest --passWithNoTests --maxWorkers 4",
+    "typecheck": "tsc --noEmit"
   },
   "author": "Grafana Labs",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@grafana/data": "9.1.0",
-    "@grafana/runtime": "9.1.0",
-    "@grafana/toolkit": "9.1.0",
-    "@grafana/ui": "9.1.0",
-    "@types/lodash": "latest",
+    "@babel/core": "^7.21.4",
+    "@grafana/e2e-selectors": "10.4.2",
+    "@grafana/eslint-config": "^7.0.0",
+    "@grafana/experimental": "1.7.11",
+    "@grafana/tsconfig": "^1.2.0-rc1",
+    "@swc/core": "^1.3.90",
+    "@swc/helpers": "^0.5.0",
+    "@swc/jest": "^0.2.26",
+    "@testing-library/jest-dom": "6.1.4",
+    "@testing-library/react": "14.0.0",
+    "@types/jest": "^29.5.0",
+    "@types/lodash": "^4.14.194",
+    "@types/node": "^20.8.7",
+    "@types/react-router-dom": "^5.2.0",
+    "@types/testing-library__jest-dom": "5.14.8",
     "aedes": "^0.45.0",
-    "cspell": "6.13.3"
-  },
-  "resolutions": {
-    "rxjs": "7.5.6"
+    "copy-webpack-plugin": "^11.0.0",
+    "cspell": "6.13.3",
+    "css-loader": "^6.7.3",
+    "eslint-plugin-deprecation": "^2.0.0",
+    "eslint-webpack-plugin": "^4.0.1",
+    "fork-ts-checker-webpack-plugin": "^8.0.0",
+    "glob": "^10.2.7",
+    "identity-obj-proxy": "3.0.0",
+    "jest": "^29.5.0",
+    "jest-environment-jsdom": "^29.5.0",
+    "prettier": "^2.8.7",
+    "replace-in-file-webpack-plugin": "^1.0.6",
+    "rxjs": "7.8.0",
+    "sass": "1.63.2",
+    "sass-loader": "13.3.1",
+    "style-loader": "3.3.3",
+    "swc-loader": "^0.2.3",
+    "ts-node": "^10.9.1",
+    "tsconfig-paths": "^4.2.0",
+    "typescript": "4.8.4",
+    "webpack": "^5.86.0",
+    "webpack-cli": "^5.1.4",
+    "webpack-livereload-plugin": "^3.0.2"
   },
   "engines": {
     "node": ">=16"
-  }
+  },
+  "dependencies": {
+    "@emotion/css": "11.10.6",
+    "@grafana/data": "10.4.2",
+    "@grafana/runtime": "10.4.2",
+    "@grafana/schema": "10.4.2",
+    "@grafana/ui": "10.4.2",
```

Comment:
since we are bumping this -- we should also increase the minimum version here:
https://github.com/grafana/mqtt-datasource/blob/main/src/plugin.json#L40